tests/fss.Tests/test_db_mysql.fs

```diff
@@ -338,39 +369,40 @@ type TestTransactions() = class
         Assert.IsTrue(conn.ExecuteScalar "SELECT COUNT(*) FROM test1" :?> int64 = 1L)
         cleanTable()
 
-    [<Test>]
+    // [<Test>]
     // Insert several rows then roll back; 
     // confirm transaction isolation
     // test trans.ExecuteScalar
     // test that trans
-    member x.Test011MultipleSimultaneousTransactions() =
-        cleanTable()
-        use trans = conn.StartTrans()
-        use trans2 = conn.StartTrans()
-        trans.InsertMany([| t1b ; t1c; t1d |]) |> ignore
-        trans2.InsertOne(t1a) |> ignore
-        // confirm that the transaction is isolated
-        Assert.IsTrue(conn.ExecuteScalar "SELECT COUNT(*) FROM test1" :?> int64 = 0L)
-
-
-        let results : test1 [] = trans.Query "select * from test1"  |> Array.ofSeq
-        Assert.IsTrue(results.Length=3)
-        Assert.IsTrue(trans.ExecuteScalar "SELECT COUNT(*) FROM test1" :?> int64 = 3L)
-
-        trans2.Commit()
-
-        // the conn only sees 1 result, because only trans2 has been committed
-        let results2: test1 [] = conn.Query "select * from test1"  |> Array.ofSeq
-        Assert.IsTrue(results2.Length=1)
-        Assert.IsTrue(conn.ExecuteScalar "SELECT COUNT(*) FROM test1" :?> int64 = 1L)
-        // trans see's 4, because READCOMMIT allows us to see the commit
-        // from trans2 as well as our own.
+    // broken
```

Comment:
Perhaps it is time to just entirely drop support for mysql?  Amyris isn't using it anymore; no idea if there are other clients of this framework.